sources/System/FileSystem/FileSystem.h

```diff
@@ -12,6 +12,10 @@
 
 #define MAX_FILENAME_SIZE 256
 
+// temporary limits due to limited ram on RP2040  
+#define PICO_MAX_FILENAME_LEN 32
+#define PICO_MAX_FILE_COUNT 25
+
```

Comment:
I think this should be in picoTrackerFileSystem.h instead. Shouldn't the number be 26 if we are using the last entry to indicate that no more files can be displayed?

Reply:
yes, doh! good point!

Reply:
With placing it in `FileSystem.h` rather than `picoTrackerFileSystem.h` I did that so that I could use the constant without needing to include a the pico specific `picoTrackerFileSystem.h` in `ImportSampleDialog.cpp` to keep the code compilable for MacOS and Linux builds, but I forgot about putting the code in `ImportSampleDialog.cpp` with in `#ifdef` so I'll pix that up.